Source/Initialization/WarpXInitData.cpp

```diff
@@ -454,6 +454,10 @@ WarpX::InitLevelData (int lev, Real /*time*/)
        }
     }
 
+    if (M_ext_grid_s == "parse_m_ext_grid_function") {
+        Abort("WarpXInitData: M field initializtion parser not implemented yet");
```

Comment:
typo initialization

Reply:
fixed in 2b93f8e980890f45d7b8d7c9c6321af89706c4d1